src/frontend/src/lib/types/token.ts

```diff
@@ -71,6 +71,7 @@ export type OptionTokenUi = Option<TokenUi>;
 
 export interface TokenGroupUi {
 	header: TokenMetadata;
+	nativeToken: TokenUi;
```

Comment:
We don't need `nativeNetwork` anymore then? We add a TODO to remove it?

Reply:
Likewise, do we still need `header` afterwards? TODO to be removed?

Reply:
i would remove it afterwards, but YES we don't need them anymore. Adding a TODO